app/scripts/models/auth_brokers/base.js

```diff
@@ -406,5 +406,8 @@ define(function (require, exports, module) {
     SearchParamMixin
   );
 
-  module.exports = BaseAuthenticationBroker;
+  module.exports = {
+    Constructor: BaseAuthenticationBroker,
+    options: [ 'relier', 'window' ]
```

Comment:
style nit: this introduces an inconsistent code base style of spaces within array brackets.

We might need to look into validating this with ESLint to avoid this. Same happens with tests below.